src/test/movie.test.js

```diff
@@ -0,0 +1,35 @@
+import { screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { renderWithProviders } from './utils'
+import App from '../App'
+
+it('movies starred and saved to watch later', async () => {
+    renderWithProviders(<App />)
+
+    await userEvent.type(screen.getByTestId('search-movies'), 'forrest gump')
+    await waitFor(() => {
```

Comment:
waitFor should be only in the case we expected a change over time beyond searching 'Through the Eyes of Forrest Gump' since is a static movie and given that case it should be removed.

Note: This goes for all parts in this file where wairFor is being used